src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ImporterLocationService.cs

```diff
@@ -268,8 +268,36 @@ public class ImporterLocationService : BaseImporterService
             predicateBuilder = predicateBuilder
                 .And(location => location.PlanningArea_Code == (planningArea != null ? planningArea.Code : null)
                                  && location.PlanningArea_Name == (planningArea != null ? planningArea.Name : null));
-            
-            return context.Location.AsNoTracking().FirstOrDefault(predicateBuilder);
+
+            // Can return multiple locations because SQL comparison is case insensitive
+            var locations = context.Location.AsNoTracking()
+                .Where(predicateBuilder)
+                .ToList();
+
+            foreach(var location in locations)
```

Comment:
You can use LINQ syntax here, replacing the whole `foreach(...) { if(...) { return location } return null }`
with `return locations.FirstOrDefault(...);`